src/System.Text.Json/tests/Serialization/Stream.ReadTests.cs

```diff
@@ -79,5 +81,17 @@ public static async Task ReadPrimitivesWithTrailingTriviaAsync()
                 Assert.Equal(1, i);
             }
         }
+
+        [Fact]
+        public static async Task ReadCollectionPassingNullValue()
+        {
+            using (MemoryStream stream = new MemoryStream(Encoding.UTF8.GetBytes("null")))
+            {
+                JsonSerializerOptions options = new JsonSerializerOptions();
+
+                var collection  = await JsonSerializer.ReadAsync<IList<object>>(stream, options);
```

Comment:
nit: Don't use `var` here. Generally, we only use `var` when it can be statically inferred from the right hand side what the type was (for instance if the `new` keyword was used or an explicit cast was there making it obvious). It helps reviewers/reading the code.

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style

```suggestion
                IList<object> collection  = await JsonSerializer.ReadAsync<IList<object>>(stream, options);
```